fabric-rendering-v0/src/main/java/net/fabricmc/fabric/api/client/event/HudRenderCallback.java

```diff
@@ -0,0 +1,20 @@
+package net.fabricmc.fabric.api.client.event;
+
+import net.fabricmc.fabric.api.event.Event;
+import net.fabricmc.fabric.api.event.EventFactory;
+
+public interface HudRenderCallback {
+	Event<HudRenderCallback> EVENT = EventFactory.createArrayBacked(HudRenderCallback.class, (listeners) ->
+			(delta) -> {
+		for (HudRenderCallback event : listeners) {
```

Comment:
What's going on with the indentation here?

Reply:
I looked at https://github.com/FabricMC/fabric/blob/cd4279539f1bdd8f39fbad7b7339d7f1f9d4c07e/fabric-events-lifecycle-v0/src/main/java/net/fabricmc/fabric/api/event/client/ItemTooltipCallback.java#L31